tests/forms_tests/tests/test_forms.py

```diff
@@ -2887,6 +2887,36 @@ class DateAgeForm(Form):
         form = DateAgeForm(data, initial={"date_age": ["200-10-10", 14]})
         self.assertTrue(form.has_changed())
 
+    def test_clean_disabled_multivalue(self):
+        """
+        #28882 -- Fixed bug with saving disabled MultiValueField
+        """
+        class RatioWidget(MultiWidget):
+
+            def __init__(self):
+                widgets = (NumberInput, NumberInput)
+                super().__init__(widgets)
+
+            def decompress(self, value):
+                return [int(x) for x in value.split(':')]
+
+        class RatioField(MultiValueField):
+            widget = RatioWidget
+
+            def __init__(self, **kwargs):
+                fields = (IntegerField(), IntegerField())
+                super().__init__(fields, **kwargs)
+
+            def compress(self, data_list):
+                return ':'.join(str(x) for x in data_list)
+
+        class RatioForm(Form):
+            ratio = RatioField(disabled=True)
+
+        form = RatioForm({}, initial={'ratio': '16:9'})
+        form.full_clean()
+        self.assertEqual(form.errors, {})
```

Comment:
I wonder if some assertion about the correct values should be added. If I change the `value = ` line in the fix to `value = [123, 125]` (some random, hardcoded values), the test still passes.